Source/DafnyRuntime/DafnyRuntimeJava/src/main/java/dafny/Helpers.java

```diff
@@ -213,7 +213,7 @@ public static void withHaltHandling(Runnable runnable) {
         try {
             runnable.run();
         } catch (DafnyHaltException e) {
-            System.err.println("[Program halted] " + e.getMessage());
+            System.out.println("[Program halted] " + e.getMessage());
```

Comment:
`err` sounds right; why change it to `out`?

Reply:
! Ok to merge this as is, but can we maybe open an issue to debate this?  @robin-aws , was it on purpose that `expect` failure are printed to `stdout`?